utils/cache.py

```
@@ -16,20 +16,30 @@
     This class provides a comprehensive metrics tracking system for monitoring various application events and performance indicators.
-    It uses Prometheus-style counters and histograms to record submission attempts, cache interactions, and document processing metrics.
+    It uses Prometheus-style counters and histograms to record submission attempts, cache interactions metrics.
 
     Attributes:
         NAMESPACE (str): The base namespace for all metrics in the web application.
     """
 
     NAMESPACE = "dozens"
+    _instance = None
+
+    def __new__(cls):
```

Ruff (0.11.9)

Missing return type annotation for special method `__new__`

(ANN204)